aip/general/0140.md

```diff
@@ -88,6 +88,31 @@ For uniformity, field names that contain both a noun and an adjective
 - `collected_items` (**not** `items_collected`)
 - `imported_objects` (**not** `objects_imported`)
 
+### Verbs
+
+Field names **must not** be expressed to reflect an intent or action.
+That is, they **must not** be expressed as verbs. Rather, the field name
+should express the condition of that aspect of the resource as defined
+by the value of the field in question.
+
+When performing a mutation, the field value is set to the desired
+*final* value of that property. When performing a read, the field value
+is returned as the *current* value of that property. As such, the name
+should always be *a noun or an adjective* as it describes what is so,
+not what to do.
+
+- `collected_items` (**not** `collect_items`)
+- `imported_objects` (**not** `import_objects`)
```

Comment:
Can we include a boolean example? These are by far the worst culprits in API review
```suggestion
- `imported_objects` (**not** `import_objects`)
- `extra_logging_enabled` (**not** `enable_extra_logging`)
```

Reply:
Agreed, good one! Fixing.